package.json

```diff
@@ -62,21 +63,22 @@
     "i18n-js": "^3.0.0"
```

Comment:
do we need those all updates for devDependencies?

Reply:
no